.github/README.md

```diff
@@ -17,7 +17,11 @@ Please note that while the issue tracker in this Github project can be a conveni
 
 ## Getting Started
 
-Full instruction can be found at https://wiki.openjdk.java.net/display/OpenJFX/Building+OpenJFX
+* [JavaFX Official Website](https://openjfx.io/)
```

Comment:
This site is mainly for contributors to JavaFX itself, so just pointing application developers to the [openjfx.io](https://openjfx.io/) community site (which should called that and not the _JavaFX official site_) should suffice. I don't think the other links you want to add are needed, since they can be reached from `openjfx.io`. 

I propose something more like the following for an updated "Getting Started" section:

----------

## Getting Started

For instructions on how to build JavaFX, see the [Building OpenJFX](https://wiki.openjdk.java.net/display/OpenJFX/Building+OpenJFX) Wiki page.

For information about downloading and using JavaFX, see the [JavaFX community site](https://openjfx.io/).